automation_tools/__init__.py

```diff
@@ -1514,3 +1441,29 @@ def update_packages(*args, **kwargs):
         quiet=kwargs.get('quiet', False),
         warn_only=kwargs.get('warn_only', False),
     )
+
+
+# =============================================================================
+# Helper functions
+# =============================================================================
+def manage_daemon(action, daemon, pty=True, warn_only=False):
+    """Manage a system daemon
+
+    :param str action: Daemon action like start, stop, restart, info
+    :param str daemon: Daemon name to perform the action
+    :param bool pty: Controls the creation of a pseudo-terminal when managing
+        the daemon. Some daemons actions fail with pty=True.
+    :param bool warn_only: Will be passed directly to Fabric's run
+
+    """
+    if distro_info()[1] >= 7:
+        command = 'systemctl {} {}'.format(action, daemon)
```

Comment:
@Ichimonji10 was a trailing `,` in this line. Have copied more than needed.

Thanks for catching this